assets/terraform-modules/bootkube/versions.tf

```diff
@@ -5,15 +5,15 @@ terraform {
   required_providers {
     local = {
       source  = "hashicorp/local"
-      version = "2.0.0"
+      version = "2.1.0"
     }
     template = {
       source  = "hashicorp/template"
       version = "2.2.0"
     }
     tls = {
       source  = "hashicorp/tls"
-      version = "2.2.0"
+      version = "3.1.0"
```

Comment:
This commit mentions only about Local but also updates TLS.

Reply:
Aha forgot that, fixed it!